packages/cli/src/seeder-daemon

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const { SDK } = require('@geut/permanent-services')
+
+;(async () => {
```

Comment:
I think we should a `catch` clause. This will help if something would wrong otherwise we could miss it.
Something like this: 
```js
const daemon = async () => { ... }
daemon().catch(err => { 
  console.error(err) 
  process.exit(1)
}
```